generator/graphql-kotlin-schema-generator/src/main/kotlin/com/expediagroup/graphql/generator/internal/extensions/kClassExtensions.kt

```diff
@@ -88,8 +89,19 @@ internal fun KClass<*>.getSimpleName(isInputClass: Boolean = false): String {
         ?: this.simpleName
         ?: throw CouldNotGetNameOfKClassException(this)
 
+    fun Annotation.isInputOnlyAnnotation(): Boolean = when (this) {
```

Comment:
lets make it internal and move it to a top level function

Reply:
I made it top-level, private and decided to make it an extension of `KClass` rather than `Annotation`.
What do you think?